ProjectSettings/ProjectSettings.asset

```diff
@@ -160,10 +167,10 @@ PlayerSettings:
     serializedVersion: 2
     m_Bits: 238
   iPhoneSdkVersion: 988
-  iOSTargetOSVersionString: 
+  iOSTargetOSVersionString: 7.0
   tvOSSdkVersion: 0
   tvOSRequireExtendedGameController: 0
-  tvOSTargetOSVersionString: 
+  tvOSTargetOSVersionString: 9.0
```

Comment:
I'm not sure if this is in your project settings, but this program doesn't need to build to xbox, iOS, tvOS...

Reply:
I didn't actively change it

Reply:
in that case I think it shouldn't have been committed then

Reply:
which is why using a friendly UI (like github for desktop or github in unity) might be really helpful